docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -388,3 +422,970 @@ GET /api/proposals
+  Splinter daemon.
+
+* `status` identifies the current status of the service. In this case, it is set
+  to `pending` because the proposal to create the gameroom has not yet been
+  accepted.
+
+* `created_time` is when the gameroom service entry was introduced in the table.
+
+* `updated_time` is when the gameroom service entry was last updated.
+
+At the end of the operation, the `gameroom_service` table looks like this:
+
+| `id` | `circuit_id` |
+| :--- | :--- |
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+
+
+| `service_id` | `service_type` | `allowed_nodes` |
+| :--- | :--- | :--- |
+| `gameroom_acme-node-000` | `scabbard` | `{"acme-node-000"}` |
+| `gameroom_bubba-node-000` | `scabbard` | `{"bubba-node-000"}` |
+
+
+| `arguments` | `status` |
+| :--- | :--- |
+|`"peer_services": [ "gameroom_bubba-node-000" ], "admin_keys": ....`|`pending`|
+|`"peer_services": [ "gameroom_bubba-node-000" ], "admin_keys": ....`|`pending`|
+
+
+| `created_time` | `updated_time` |
+| :--- | :--- |
+|`<time entry was created>`|`<time entry was updated>`|
+|`<time entry was created>`|`<time entry was updated>`|
+
+##### I-2.8.3. New `gameroom_service` table entry
+
+The Gameroom application authorization handler adds a new entry to the
+`gameroom_proposal` table, which contains the information about the gameroom
+proposal.
+
+``` sql
+CREATE TABLE IF NOT EXISTS  gameroom_proposal (
+  id BIGSERIAL PRIMARY KEY,
+  proposal_type TEXT NOT NULL,
+  circuit_id TEXT NOT NULL,
+  circuit_hash TEXT NOT NULL,
+  requester TEXT NOT NULL,
+  requester_node_id TEXT NOT NULL,
+  status TEXT NOT NULL,
+  created_time TIMESTAMP NOT NULL,
+  updated_time TIMESTAMP NOT NULL,
+  FOREIGN KEY (circuit_id) REFERENCES gameroom(circuit_id) ON DELETE CASCADE
+);
+```
+
+* `circuit_id`, `proposal_type`, `circuit_hash`, `requester` and
+  `requester_node_id` are extracted directly from the circuit proposal message
+  received from the Splinter daemon.
+
+* `status` identifies the current status of the proposal. In this case, it is
+  set to `pending` because the proposal to create the gameroom has not yet been
+  accepted.
+
+* `created_time` is when the gameroom proposal entry was introduced in the table.
+
+* `updated_time` is when the gameroom proposal entry was last updated.
+
+At the end of the operation, the `​gameroom_proposal​` table looks like this:
+
+| `id` | `circuit_id` |
+| :--- | :--- |
+|`<auto generated id>`|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|
+
+
+| `proposal_type` | `circuit_hash` | `requester` | `requester_node_id` |
+| :--- | :--- | :--- | :--- |
+|`Create`|`<hash of circuit definition>`|`<public key of requester>`|`acme-node-000`|
+
+
+| `status` | `created_time` | `updated_time` |
+| :--- | :--- | :--- |
+| `pending` | `<time entry was created>` | `<time entry was updated>` |
+
+##### I-2.8.3. New `gameroom_notification` table entry
```

Comment:
Wrong number and title: It should be I-2.3.5 (and mention gameroom_notification).